internal/handlers/index_test.go

```diff
@@ -0,0 +1,131 @@
+package handlers
+
+import (
+	"net/http/httptest"
+	"testing"
+
+	"github.com/gofiber/fiber/v2"
+	"github.com/jiotv-go/jiotv_go/v3/internal/config"
+	"github.com/jiotv-go/jiotv_go/v3/pkg/television"
+)
+
+func TestIndexHandlerWithDefaultConfig(t *testing.T) {
+	// Save original config
+	originalCfg := config.Cfg
```

Comment:
![medium](https://www.gstatic.com/codereviewagent/medium-priority.svg)

For restoring global state in tests, `t.Cleanup` is preferred over a manual restore at the end of the test function. It's more robust as it's guaranteed to run even if the test panics or is aborted with `t.Fatal`.

You can add `t.Cleanup(func() { config.Cfg = originalCfg })` right after `originalCfg` is defined at the top of the test function and remove the manual restore at line 130.